randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -1006,7 +1021,17 @@ Extra - room_id: [30]
   * Extra - blockx: 19
   * Extra - blocky: 8
   > Door to Twin Caverns East (Upper)
-      Movement (Intermediate)
+      All of the following:
+          Movement (Intermediate)
+          Any of the following:
+              # With WJ: https://youtu.be/FVO9oTKMA3c
+              Wall Jump (Intermediate)
```

Comment:
I feel like this WJ should be bumped. you have to time a walljump slightly before/during the item transition, and if you fail you possibly cannot go back up.